src/components/bulk-email-tool/bulk-email-form/BuildEmailFormExtensible/index.jsx

```diff
@@ -0,0 +1,122 @@
+import { useContext } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import useDeepCompareEffect from 'use-deep-compare-effect';
+import {
+  Form,
+  Spinner,
+  useToggle,
+} from '@openedx/paragon';
+import { BulkEmailContext } from '../../bulk-email-context';
+import useMobileResponsive from '../../../../utils/useMobileResponsive';
+import PluggableComponent from '../../../PluggableComponent';
+
+import { withContextProvider, useDispatch } from './context';
+import { actionCreators as formActions } from './context/reducer';
+
+const BuildEmailFormExtensible = ({ courseId, cohorts, courseModes }) => {
```

Comment:
Question about naming here: since this is completely replacing the current `BulkEmailForm`, does it make sense to simply use that name instead of using a new name? If not, and `BulkEmailForm` doesn't feel like an accurate name anymore, then I feel the `Extensible` is probably not needed and the name could simply be `BuildEmailForm`

Reply:
That makes sense I changed the name for `BulkEmailForm` as was